src/main/kotlin/no/nav/persondataapi/rest/inntektController.kt

```diff
@@ -11,18 +11,13 @@ import org.springframework.web.bind.annotation.RestController
 
 @RestController
 class inntektController(
-    private val tokenValidationContextHolder: TokenValidationContextHolder,
     private val inntektClient: InntektClient
 ) {
     @GetMapping("/inntekt")
```

Comment:
Jeg var litt usikker på hva disse var for – var det for å hente ut prod-data fra kildesystemene? I så fall bør vi kanskje vurdere å bare fjerne dem. 

Enn så lenge fjernet jeg token-valideringen, siden den ikke var i bruk uansett.